src/Toolkit/Dom.php

```diff
@@ -502,12 +495,12 @@ public static function remove(DOMNode $node): void
 	/**
 	 * Executes an XPath query in the document
 	 *
-	 * @param string $query
 	 * @param \DOMNode|null $node Optional context node for relative queries
-	 * @return \DOMNodeList|false
 	 */
-	public function query(string $query, ?DOMNode $node = null)
-	{
+	public function query(
+		string $query,
+		DOMNode $node = null
```

Comment:
```suggestion
		DOMNode|null $node = null
```

Reply:
@bastianallgeier @afbora was this really needed? From my view it wouldn't make much sense to support actively passing `null` here, or does it? Only to leave it optional (so `= null` but when passing something it shouldn't be null).